src/vl-rich-data-table.js

```diff
@@ -38,15 +39,27 @@ export class VlRichDataTable extends VlElement(HTMLElement) {
         return 'vl-rich-data-table';
     }
 
+    static get _defaultSearchColumnSize() {
+        return 4;
+    }
+
     constructor() {
         super(`
             <style>
                 @import "/src/style.css";
+                @import "/node_modules/vl-ui-icon/dist/style.css";
+                @import "/node_modules/vl-ui-button/dist/style.css";
                 @import "/node_modules/vl-ui-data-table/dist/style.css";
             </style>
             <div is="vl-grid" is-stacked>
+                <div is="vl-column" id="toggle-filter" class="vl-u-align-right" hidden size="12">
+                    <button id="toggle-filter-button" is="vl-button-link" type="button" aria-label="Toon de filter"><span is="vl-icon" icon="content-filter" before></span>Filter</button>
+                </div>
                 <div id="search" is="vl-column" size="0">
-                    
+                    <button id="close-filter-button" class="vl-filter__close" hidden type="button">
+                        <span is="vl-icon" icon="close"></span>
+                        <span class="vl-u-visually-hidden">Filter sluiten</span>
```

Comment:
tekst niet configureerbaar?

Reply:
i18n?

Reply:
Inderdaad best configureerbaar maken, meestal voorzien we voor dergelijke teksten zowel tekst attribuut als eens slot optie (voor vl-ui-proza-message).

Reply:
Zowel attributen als slots? Enkel slots lijkt me toch ook voldoende niet?

Reply:
Slots covert zeker de lading. Extra attributen voorzien is een kleine technische uitbreiding die handig kan zijn voor afnemers die geen complexe HTML willen meegeven. Tot op vandaag hebben we overal beide voorzien, maar misschien is het hier niet nodig.